packages/@okta/vuepress-site/docs/api/resources/oidc/index.md

```diff
@@ -568,7 +570,7 @@ Content-Type: application/json;charset=UTF-8
 #### Key Rotation
 The keys that are used to sign tokens are periodically changed. Okta automatically rotates your authorization server's keys on a regular basis.
 
-Clients can opt-out of automatic key rotation by changing the client sign-in mode. In this case, passing the `client_id` with your request retrieves the keys for that specific client.
+Clients can opt-out of automatic key rotation by changing the client sign-in mode for Okta Org Authorization Server. In this case, passing the `client_id` with your request retrieves the keys for that specific client.
```

Comment:
add the word "the" in one place:

Clients can opt-out of automatic key rotation by changing the client sign-in mode for the Okta Org Authorization Server. In this case, passing the `client_id` with your request retrieves the keys for that specific client.

Reply:
Made the edit.